README.md

```diff
@@ -174,6 +174,12 @@ This example shows how to enable the `data`, `orgs`, and `metadata` toolsets whe
      }
 ```
 
+#### Dynamic Tools
+
+The `--dynamic-tools` flag enables dynamic tool discovery and loading. When this flag is set, the MCP server starts with a minimal set of core tools and will load new tools as the need arises. This is useful for reducing initial context size and improving LLM performance.
+
+**NOTE:** This feature may not work across all environments.
```

Comment:
Are we able to state which envs are known to work and all others are unknown just to help users decide?